cpinstaller/src/boot.py

```diff
@@ -0,0 +1,3 @@
+import storage
+
+storage.disable_usb_drive()
```

Comment:
Is this ever copied over?

Reply:
Not yet. I was trying to get something up for now since trying to disable the boot.py was ending up being more complicated than I thought, but that was planned to be part of another update.